pkg/webui/console/store/middleware/logics/events.js

```diff
@@ -104,12 +105,33 @@ const createEventsConnectLogics = function(reducerName, entityName, onEventsStar
       async process({ getState, action }, dispatch) {
         const { id } = action
 
+        const {
+          addToBuffer: addToEventBuffer,
+          clearBuffer: clearEventBuffer,
+        } = createBufferedProcess(events => {
+          const processedEvents = events
+            .filter(
+              // See https://github.com/TheThingsNetwork/lorawan-stack/pull/2989
+              event => event.name !== 'events.stream.start' && event.name !== 'events.stream.stop',
+            )
+            .sort((a, b) => (a.time > b.time ? -1 : 1))
+
+          if (processedEvents.length > 0) {
```

Comment:
This one is to check whether there is any event left after removing `events.stream.start/stop`. Otherwise, there should be no empty event arrays arriving here.